tests/unittests/tests-clif/tests-clif.c

```diff
@@ -313,6 +313,31 @@ static void test_clif_get_attr_empty(void)
     TEST_ASSERT_EQUAL_INT(CLIF_NOT_FOUND, r);
 }
 
+static void tests_clif_decode_encode_minimal(void)
+{
+    #define BUFF_SIZE 50
```

Comment:
Maybe move this to the top, I somehow don't think we have `#define`s in the functions, even though I understand the scope...